<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>北邮人银行</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics/buttonIcon/buttonicon.qrc">
    <normaloff>:/Bank_202px_1194726_easyicon.net.png</normaloff>:/Bank_202px_1194726_easyicon.net.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢迎</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>951</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="cardList">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>611</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string>卡号</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>所属银行</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>所有者姓名</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>所有者联系方式</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>银行卡信息</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>31</y>
       <width>191</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>卡号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cardID">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>银行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bankName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>所有者</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ownerName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>身份证</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ownerID">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>手机号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ownerPhone">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>余额</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="money">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="goback">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="history">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询历史记录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>113</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2">
     <item>
      <widget class="QPushButton" name="editPassword">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改登录密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerNewCard">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>办理银行卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="storeMoney">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>存款</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="takeMoney">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取款</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCard">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>撤销银行卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editPayCode">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改支付密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="diaryList">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>422</width>
      <height>381</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>日期</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>操作</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>金额</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <zorder>diaryList</zorder>
   <zorder>layoutWidget1</zorder>
   <zorder>welcomeLabel</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>logout</zorder>
   <zorder>cardList</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pics/buttonIcon/buttonicon.qrc"/>
 </resources>
 <connections/>
</ui>
